docker-compose/nginx/http.conf

```diff
@@ -5,3 +5,24 @@ server {
 
 	include shanoir.conf;
 }
+
+server {
+	listen		80;
+	server_name	SHANOIR_VIEWER_OHIF_URL_HOST;
+
+	proxy_read_timeout 	5000s;
+
+	# increase the buffer size (the default 4kb is occasionally too small for keycloak)
+	proxy_buffer_size	8k;
+	
+	# do not buffer whole responses
+	proxy_buffering 	off;
+	
+	proxy_set_header	Host $http_host;
+	proxy_set_header	X-Forwarded-For $remote_addr;
```

Comment:
`X-Forwarded-For` and `X-Forwarded-Proto` must be generated only if `SHANOIR_X_FORWARDED=generate`
(in production they are generated by the frontal nginx server)

I fixed it + deduplicated the ohif-viewer config + simplified the entrypoint in michaelkain/shanoir-ng#8

Reply:
Great Anthony, thank you for your help here, what makes it much clearer!